<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>*Время выполнения алгоритма  указано в мс</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>534</width>
      <height>361</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; background-color:lightgrey;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>11</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>47</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>90</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>n</string>
     </property>
     <property name="background">
      <color>
       <red>173</red>
       <green>163</green>
       <blue>253</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>QuickSort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BubbleSort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SelectSort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>InsertSort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HeapSort</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>118</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_qs">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Quick Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_bs">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Bubble Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_ss">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Selecton Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_is">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Insertion Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_hs">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Heap Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="vgeneration">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black; background-color: grey;</string>
       </property>
       <property name="text">
        <string>Cгенерировать вектор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file_read">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black; background-color: grey;</string>
       </property>
       <property name="text">
        <string>Ввести вектор из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculate">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black; background-color: grey;</string>
       </property>
       <property name="text">
        <string>Расчет результатов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_res">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black; background-color: grey;</string>
       </property>
       <property name="text">
        <string>Сохранить результаты  в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_graphics">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black; background-color: grey;</string>
       </property>
       <property name="text">
        <string>Показать графики</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black; background-color: grey;</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: white;</string>
    </property>
    <property name="text">
     <string>Добро пожаловать!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
